### PR TITLE
fix(flow): filter out all ALTER_SCHEDULE flow instances in 'all tickets', 'created by me' and 'approving by me'

### PR DESCRIPTION
**Pull request by MarkPotato777**
#### What type of PR is this?

type-bug
#### What this PR does / why we need it:
Filter out all ALTER_SCHEDULE flow instances in 'all tickets', 'created by me' and 'approving by me'.
The flow instance that created by schedule should be not seen in the tickets. If users want to view them, go to the schedule module.

#### Which issue(s) this PR fixes:

Fixes #

#### Additional documentation e.g., usage docs, etc.:

```docs

```

### CODE REVIEW

File: server/odc-service/src/main/java/com/oceanbase/odc/service/flow/FlowInstanceService.java [hunk 1]
```diff
@@ -471,7 +471,6 @@ public Page<FlowInstanceEntity> listAll(@NotNull Pageable pageable, @NotNull Que
                     TaskType.SHADOWTABLE_SYNC,
                     TaskType.PARTITION_PLAN,
                     TaskType.ONLINE_SCHEMA_CHANGE,
-                    TaskType.ALTER_SCHEDULE,
```
Comment (yhilmare) [inline server/odc-service/src/main/java/com/oceanbase/odc/service/flow/FlowInstanceService.java:474]: @guowl3 may check this

Comment (yhilmare) [reply server/odc-service/src/main/java/com/oceanbase/odc/service/flow/FlowInstanceService.java:474]: how can we process alter schedule ticket if you filter out them?

Comment (MarkPotato777) [reply server/odc-service/src/main/java/com/oceanbase/odc/service/flow/FlowInstanceService.java:474]: Users need to go to the schedule module to view the tickets.

**Review by yizhouxw — APPROVED** [review]
lgtm

**Pull request merged**